testsuite/tests/kuadrant/authorino/operator/tls/conftest.py

```diff
@@ -8,12 +9,19 @@
 
 
 @pytest.fixture(scope="session")
-def certificates(cfssl, authorino_domain, wildcard_domain):
+def cert_attributes_custom() -> Dict[str, str]:
+    """Certificate attributes that are different from the default ones"""
+    return {k: f"{v}-other" for k, v in CFSSLClient.DEFAULT_NAMES[0].items()}
```

Comment:
I would rather copy this or create a minimalistic version of this dict, I do not like idea of depending on the defaults